Jenkinsfile

```diff
@@ -185,30 +187,50 @@ pipeline {
       steps {
         sh "sed 's/__IMAGE_TAG__/${GIT_COMMIT}/g' kubernetes/deployment-production.tmpl | kubectl --context azure apply --record -f -"
       }
-      post {
-        success {
-          script {
-            if (env.TAG_NAME == 'production-release') {
-              slackSend (
-                color: '#00FF00',
-                message: "SUCCESSFUL: Job '${env.JOB_NAME} [${env.BUILD_NUMBER}]' (${env.BUILD_URL})",
-                channel: "#ops"
-              )
-            }
-          }
-        }
+    }
 
-        failure {
-          script {
-            if (env.TAG_NAME == 'production-release') {
-              slackSend (
-                color: '#FF0000',
-                message: "FAILED: Job '${env.JOB_NAME} [${env.BUILD_NUMBER}]' (${env.BUILD_URL})",
-                channel: "#ops"
-              )
-            }
-          }
-        }
+    stage('Migrate production database') {
+      when { tag 'production-migrate' }
+      agent any
+      steps {
+        sh """
+          sed 's/__IMAGE_TAG__/${GIT_COMMIT}/g' kubernetes/job-db-migrate-production.tmpl | kubectl --context azure apply --record -f -
+
+          kubectl wait --for=condition=complete --timeout=1200s job/panoptes-migrate-db-production-${GIT_COMMIT}
```

Comment:
I was going to try the job TTL instead of the timeout here but either should work, https://kubernetes.io/docs/concepts/workloads/controllers/job/#ttl-mechanism-for-finished-jobs

if we set a day or so for the TTL i think that'll help with the cleanup code here too.